util/src/builder/mod.rs

```diff
@@ -1,7 +1,9 @@
 //! Builders for large structs.
 #![allow(clippy::module_name_repetitions)]
 
+mod button;
 mod callback_data;
 pub mod command;
 
+pub use self::button::{ButtonBuilder, ButtonError, ButtonErrorType};
 pub use self::callback_data::CallbackDataBuilder;
```

Comment:
After thinking about it, it would be even better to export the whole `button` module, as it contains several related types.

```suggestion
pub mod button;
mod callback_data;
pub mod command;

pub use self::callback_data::CallbackDataBuilder;
```

Documentation examples should be updated accordingly.